src/api/getWilayaByZipCode.js

```diff
@@ -0,0 +1,21 @@
+const projectWilaya = require('../utils/projections/wilayaProjection');
+
+const getWilayaByZipCode = data =>
+/**
+   * Takes a zip code (postal code) and returns it's wilaya.
+   *
+   * @example <p> Get 1000 wilaya
+   *
+   * //returns {mattricule: '1', name: 'Adrar', ...} because the zip code 1000 belongs to Adrar
+   * getWilayaByZipCode(31)
+   *
+   * @param { Number } zipCode postal code
```

Comment:
@abachi thank you! Just one more change: Update the function documentation here:

```suggestion
   * @param { Number } zipCode postal code
   * @param {String[]} projection a list of  wilaya object attributes to keep
```

And in the README file